app/experimenter/nimbus-ui/templates/nimbus-ui.html

```diff
@@ -0,0 +1,28 @@
+{% load static %}
```

Comment:
Oh good yes, the only thing I'd maybe suggest is instead of having it in the template root, putting it in a subpath so referring to it later in views is less ambiguous, like 

`templates/nimbus/index.html`  

so the view would say

`template_name = "nimbus/index.html"`

But that's just a suggestion 👍

Reply:
I can dig it :)